src/node/internal/internal_assert.ts

```diff
@@ -65,7 +65,7 @@ function createAssertionError(
   return error;
 }
 
-function assert(actual: unknown, message?: string | Error): asserts actual {
+export function assert(actual: unknown, message?: string | Error): asserts actual {
```

Comment:
This ends up being exported in the public api also. We shouldn't export this here. What you want is to import the `ok` method (note that just below `export const ok = assert`.